### PR TITLE
Set language to en_US to fix issues with gcc.

### PR DESCRIPTION
**Pull request by jobermayr**
Fixes: #2166 once for all

### CODE REVIEW

**Comment by jobermayr** [comment]
`$ gradle buildGhidra`
`$ LC_MESSAGES=de gradle buildGhidra`
Both work without any issues with this patch.

File: build.gradle [hunk 1]
```diff
@@ -19,6 +19,16 @@ apply from: 'gradle/root/eclipse.gradle'
 apply from: "gradle/support/eclipseLauncher.gradle"
 apply from: "gradle/support/loadApplicationProperties.gradle"
 
+import java.lang.reflect.Field;
+
+public static void updateEnv(String name, String val) throws ReflectiveOperationException {
+    Map<String, String> env = System.getenv();
+    Field field = env.getClass().getDeclaredField("m");
```
Comment (ryanmkurtz) [inline build.gradle:26]: What is this line doing with the "m" field?

Comment (jobermayr) [reply build.gradle:26]: https://blog.quirk.es/2009/11/setting-environment-variables-in-java.html?m=1
"By accessing that m we can change the JVM's copy of the environment from under its very nose. "

Comment (ryanmkurtz) [reply build.gradle:26]: For me, the type of `env` is `class org.gradle.internal.classpath.AccessTrackingEnvMap`, not `java.util.Collections.UnmodifiableMap`.  That is why it's failing for me.

**Pull request closed without merging**